pkg/reconciler/route/resources/ingress.go

```diff
@@ -97,6 +105,8 @@ func MakeIngressSpec(
 	// Sort the names to give things a deterministic ordering.
 	sort.Strings(names)
```

Comment:
Tag names are sorted by this part. So the precedence between tag based routing rules is determined here and it follows the sorting rule of `sort.Strings`

Reply:
https://github.com/knative/serving/pull/6036/commits/2a4d4a72e98c77b9c25a9342cb225263d8a54480#r352950324